.github/pull_request_template.md

```diff
@@ -1,26 +1,19 @@
-***Delete section if irrelevant***
+Fixes #(issue)
 
-### List related PRs if any
+## Type of change
 
-- pr link
+Please delete options that are not relevant.
```

Comment:
Given the type of change is a collection of checkbox, isn't it more natural to expect the contributor to "toggle the relevant options/ checkbox"?

Reply:
That is more concise, I will remove this line